addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -377,7 +377,7 @@ export const editorCommands = {
                 const block = closestBlock(el);
                 el.style.fontWeight = isBold(block) ? 'normal' : getComputedStyle(block).fontWeight;
             } else {
-                el.style.fontWeight = 'bolder';
```

Comment:
@dmo-odoo Does this commit seem ok to you? The original specs of the task were not about changing the editor behavior but it eventually changed into this :grimacing: 

@bvr-odoo Commit message: be more explicit in the title, something like `[FIX] web_editor: use 'bold' instead of 'bolder' with the editor` maybe? It fits the length-rule and immediately explain what the commit does.

Reply:
I think `bold` makes more sense than `bolder` in this case. However, I'd like to ping @Zinston and @Goaman because I remember we had issues with this in the past that we used to track down to bootstrap css overrides in enterprise. Did we fix this bug ? Or did we ultimately divert course without fixing it and this change would finally fix it ? I think so because I don't remember us meddling with bootstrap variables overrides and such but I might remember wrongly and you might have a better memory than I do.

Reply:
I think Emilien had used `bolder` rather than `bold` for cases like a header that we might want to make bolder than it is or things like that. So I think whatever we choose to do, what matters is mostly that `isBold` is harmonized with this `bold` method so we never have a case where we think we're gonna make something bold but nothing happens. Just my two cents.

Reply:
I took a look at the W3C "spec" for inspiration... It's... _illuminating_... https://w3c.github.io/editing/docs/execCommand/#the-bold-command

> It's a similar story for similar commands (fontName, italic, etc.). Except not for strikethrough, where it just does nothing if the selection is empty. Why strikethrough? I don't know.

He doesn't seem to know much in the end :eyes: 

Anyway two things I learned from it: they use `queryCommandState` as `isBold` (@dmo-odoo @sebgeelen any particular reason we don't?), and the recommended threshold for bold is 600.

In any case yeah we need a spec.

Reply:
I'm the one who actually introduced noise in this discussion. I feared this PR might break something that we used to fix but I couldn't remember if we actually _did_ fix it in the first place. Since none of my colleague remember the fix and I tend to think that we in fact did _not_ provide a fix, I therefore don't think your PR can do more worse than good on that front. So TL;DR: `bold` is fine by me if you tested "the most common flows" like you said.